### PR TITLE
Thu/scrolls and links

### PR DESCRIPTION
**Pull request by thumn**
## Scrollable Descriptions, Studio Art, and Links
Users can now add (linkified) links to a separate section on the Create Work/New Work form. Added Studio Art to program enums. Added persistent scrollbars to artist and artwork descriptions that are longer than the max height of the div. Changed description textbox in Update Artist form from text input to text area.

### Related PRs
None.

### Migrations
Added column "links" to Works database.
db/migrate/20190405082136_add_links_to_works.rb

### Tests Performed, Edge Cases
Scrollbar appears when text extends beyond max height. Link section in artwork description does not appear when artist does not provide links.

### Screenshots
scrolls_links
Screen Shot 2019-04-05 at 2 05 14 AM
artist_scroll

CC: @by-co

### CODE REVIEW

File: app/assets/stylesheets/pages/DetailedWork.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    }
+    
+.artwork-description::-webkit-scrollbar-thumb {
+background: $magenta;  
+border-radius: 10px;
+}
+
+.artwork-description::-webkit-scrollbar-thumb:hover {
+background: $magenta;  
+}
```
Comment (by-co) [inline app/assets/stylesheets/pages/DetailedWork.scss]: Let's also add a visible track (and for .artist-description as well):

```
.artwork-description::-webkit-scrollbar-track {
    background: $snow; 
}
```

File: app/assets/stylesheets/pages/DetailedWork.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    height: 12px;
+    }
+    
+.artwork-description::-webkit-scrollbar-thumb {
+background: $magenta;  
+border-radius: 10px;
+}
+
+.artwork-description::-webkit-scrollbar-thumb:hover {
+background: $magenta;
```
Comment (by-co) [inline app/assets/stylesheets/pages/DetailedWork.scss]: You can try using a SASS function to get a different color on hover!

```suggestion
    background: darken($magenta, 10%);  
```
